src/qvimagecore.cpp

```diff
@@ -81,6 +81,17 @@ void QVImageCore::loadFile(const QString &fileName)
     QFileInfo fileInfo(sanitaryFileName);
     sanitaryFileName = fileInfo.absoluteFilePath();
 
+    if (fileInfo.isDir())
+    {
+        QString dummyFilePath = QDir::cleanPath(sanitaryFileName + QDir::separator() + "dummy.jpg");
```

Comment:
Can you write a comment explaining the use of dummy.jpg a little? I get it but just to make it more readable